src/library/FauxControl/styled.js

```diff
@@ -28,20 +28,20 @@ export const FauxControlRoot = styled('div', {
 
     '&:hover': {
       '& > div:last-child': {
-        borderColor: !disabled ? theme.FauxControl_borderColor_hover : null
+        borderColor: borderColor ? borderColor : !disabled ? theme.FauxControl_borderColor_hover : null
```

Comment:
the logic here is wrong. The `disabled` prop should take priority. if you have a disabled field it should not allow you to change the color.

```
borderColor: !disabled ? (borderColor ? borderColor : theme.FauxControl_borderColor_hover) : null
```
or something like that

Reply:
Actually, revert all of these changes for FauxControl per our discussion:

You can do:
```
<FormField
    input={TextInput}
    label="Danger"
    caption="This is help text"
    variant="danger"
    iconEnd={null}
 />
```
Note: I actually don't mind the icon. Let's leave it instead of doing iconEnd.